dist/index.html

```diff
@@ -8,12 +8,12 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
- [ ] Please as the last reviewer said the updated array of items should stored in local storage, so the user gets the correct list values after the page reloads. Make sure to reload the page to test the array is stored.
currently you save the data but when the user first reload the page they just disappear and then when the user write a new task they comes again please make sure to fix this weried behaviour

Reply:
please check i updated your instruction.